classroom/snippets/src/test/java/TestDeleteGuardian.java

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import org.junit.Assert;
+import org.junit.Test;
+
+// Unit test class for Delete Guardian classroom snippet
+public class TestDeleteGuardian {
+
+  @Test
+  public void testDeleteGuardian() throws Exception {
+    String studentId = "insert_student_id";
+    String guardianId = "insert_guardian_id";
+    DeleteGuardian.deleteGuardian(studentId, guardianId);
+
+    Assert.assertThrows("Not found", GoogleJsonResponseException.class, () ->
```

Comment:
is "Not found" the right message here? isn't it the expectation that it wouldn't be found rather than the exception?

Reply:
Yeah I wasn't sure exactly what message to include here and Not found seems be confusing. There is no need to include a message when calling assertThrows so I've removed it and just included the conditional.